src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -588,7 +589,6 @@ export class ChatWidget extends Disposable implements IChatWidget {
 			if (this.bodyDimension) {
 				this.layout(this.bodyDimension.height, this.bodyDimension.width);
```

Comment:
Can't you get right of the layout call here too since that happens in `onDidChangeHeight`?